### ISSUE TITLE
Inconsistent hyperparameter's name between map_threshold and fdr_threshold

### ISSUE THREAD

**Issue by tbng** [nistats/nistats#280]
This might a small issue but could be confusing for users that are new to statistical analysis of brain imaging.

Nistats version is `0.0.1b`. The function `nistats.thresholding.map_threshold` and `nistats.thresholding.fdr_threshold` have different hyperparameter names which serves the same purpose, in particular:
*  `map_thresholding` has parameter `level`
* `fdr_thresholding` has parameter `alpha`

in both case are used for thresholding the statistical metric under desired level.  For example if one runs the code:
```python
from nilearn import datasets
import pandas as pd
from nistats.second_level_model import SecondLevelModel
from nistats.thresholding import map_threshold
n_samples = 2
localizer_dataset = datasets.fetch_localizer_calculation_task(
    n_subjects=n_samples)
cmap_filenames = localizer_dataset.cmaps
design_matrix = pd.DataFrame([1] * n_samples, columns=['intercept'])
second_level_model = SecondLevelModel().fit(cmap_filenames, design_matrix=design_matrix)
z_map = second_level_model.compute_contrast(output_type='z_score')
thresholded_map2, threshold2 = map_threshold(z_map, level=1.05, height_control='fdr')
```
which use 'fdr' as a threshold metric, but set the variable `level` to invalid value (not in [0, 1] range). The (expected) result is a `ValueError`:
```bash
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-1-8987fe651026> in <module>
     14 
     15 # use fdr as a threshold, but set the variable `level` to invalid value (not in [0, 1] range)
---> 16 thresholded_map2, threshold2 = map_threshold(z_map, level=1.05, height_control='fdr')

~/build/dev_env_1/lib/python3.6/site-packages/nistats/thresholding.py in map_threshold(stat_img, mask_img, level, height_control, cluster_threshold)
    113     # Thresholding
    114     if height_control == 'fdr':
--> 115         threshold = fdr_threshold(stats, level)
    116     elif height_control == 'bonferroni':
    117         threshold = norm.isf(level / n_voxels)

~/build/dev_env_1/lib/python3.6/site-packages/nistats/thresholding.py in fdr_threshold(z_vals, alpha)
     26     """
     27     if alpha < 0 or alpha > 1:
---> 28         raise ValueError('alpha should be between 0 and 1')
     29     z_vals_ = - np.sort(- z_vals)
     30     p_vals = norm.sf(z_vals_)

ValueError: alpha should be between 0 and 1
```
but in fact it would be clearer to raise the error as 
```bash
ValueError: level should be between 0 and 1
``` 
Also in my opinion `alpha` is less intuitive than `level` as a hyperparameter name for the thresholding function.

**Comment by bthirion** [comment]
I get your popint, but it is tricky: the meaning of the value taken by map_threshold has different meaning depending on the `height_control` parameter: when it's `fpr` `fdr` or `bonferroni` its is a p- or q-value and should be confined to [0, 1], but otherwise, it is an arbitrary z-variate.
maybe, we should avoid this `level` argument and replace it by 2 arguments: `alpha` and `threshold`, where one of them has to be specified and will be taken into account depending on `height_control`.

Does that make sense ?

then + 1 for adding check on alpha.

**Comment by bthirion** [comment]
Solved with #320.
Closing.
